pyvo/dal/tests/test_tap.py

```diff
@@ -180,7 +180,10 @@ def create(self, request, context):
         job.jobid = newid
         if 'test_erroneus_submit.non_existent' in request.text:
             job.phase = 'ERROR'
-            job.message = 'test_erroneus_submit.non_existent not found'
+            job._errorsummary = ErrorSummary()
+            job.errorsummary.message = Message()
+            job.errorsummary.message.content =\
+                'test_erroneus_submit.non_existent not found'
```

Comment:
are these used anywhere in the tests? E.g. if I take the test changes from this PR and run them against the main branch, this doesn't trigger a failure (changes in `pyvo/io/uws/tests/test_job.py` do).

So this is somewhat unrelated and a more generic comment: I think building up complex mock classes but then don't fully use them in the tests are a bit misleading.